docs/messaging/usage/index.md

```diff
@@ -68,7 +68,12 @@ async function requestUserPermission() {
 The permissions API for iOS provides much more fine-grain control over permissions and how they're handled within your
 application. To learn more, view the advanced [iOS Permissions](/messaging/ios-permissions) documentation.
 
-On Android, you do not need to request user permission. This method can still be called on Android devices; however, and will always resolve successfully.
+On Android pre api level 33 you do not need to request user permission. This method can still be called on Android devices; however, and will always resolve successfully. For api level 33+ you will need to request the permission manually.
```

Comment:
I normally suggest people use `react-native-permissions` as it handles everything very nicely but I like the example using nothing but built-in APIs. So I added some text mentioning both, what do you think?

```suggestion
On Android pre api level 33 you do not need to request user permission. This method can still be called on Android devices; however, and will always resolve successfully. For api level 33+ you will need to request the permission manually using either the built-in react-native `PermissionsAndroid` APIs or a related module such as `react-native-permissions`
```

Reply:
looks good